pyxem/utils/io_utils.py

```diff
@@ -408,36 +434,31 @@ def _get_mib_depth(hdr_info, fp):
     return depth
 
 
-def _read_exposures(hdr_info, fp, pct_frames_to_read=0.1, mmap_mode='r'):
+def _mib_to_daskarr(hdr_info, fp, mmap_mode='r'):
     """
-    Looks into the frame times of the first 10 pct of the frames to see if they are
-    all the same (TEM) or there is a flyback (4D-STEM).
-    For this to work, the tick in the Merlin softeare to print exp time into header
-    must be selected!
+    Reads the binary mib file into a numpy memmap object and returns as dask array object
 
     Parameters
-    -------------
+    --------------
     hdr_info: dict
-        Output from _parse_hdr function
+        output from parse_hdr function
     fp: str
         MIB file name / path
-    pct_frames_to_read : float
-        Percentage of frames to read, default value 0.1
     mmap_mode: str
-        Memmpa read mode - default is 'r'
+        memmpap read mode - default is 'r'
     Returns
-    ------------
-    exp_time: list
-        List of frame exposure times
+    --------------
+    data_da: data as a dask array object
```

Comment:
after the colon you should have the object "type" only i.e. dask.array here - then on the next line give the explanation of what it is.